test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/DotnetProcessKind.cs

```diff
@@ -9,12 +9,15 @@
 
 namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
 {
-    internal enum MSBuildProcessKind
+    internal enum DotnetProcessKind
```

Comment:
Why change this? I don't get it.

Reply:
Because I'm reusing these for the shutdown command as well and that has nothing to do with MSBuild

Reply:
> that has nothing to do with MSBuild

That's exactly my point